src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingHome.java

```diff
@@ -45,6 +46,27 @@
     @NotNull
     private final List<BlockPos> bedList = new ArrayList<>();
 
+    /**
+     * Is a female citizen living here?
+     */
+    private boolean femalePresent = false;
+
+    /**
+     * Is a male citizen living here?
+     */
+    private boolean malePresent = false;
+
+    /**
+     * Interval at which the childen are created, in ticks.
+     * Every 20 min it tries to spawn a child, 20min*60s*20ticks
+     */
+    private int childCreationInterval = 2 * 60 * 20;
+
+    /**
+     * The timer counting ticks to the next time creating a child
+     */
+    private int childCreationTimer = 0;
```

Comment:
can't we make it on average x ticks by making it a random calc % x ?

Reply:
ye that works aswell, gonna change it